antiphishing/antiphishing.py

```
@@ -345,13 +185,329 @@ async def action(self, ctx: Context, action: str):
+            "timeout": self._timeout_action
+        }
+        if action in action_methods:
+            await action_methods[action](message, redirect_chain)
+
+    async def _notify_action(self, message: discord.Message, redirect_chain: List[str]):
+        if message.channel.permissions_for(message.guild.me).send_messages:
+            with contextlib.suppress(discord.NotFound):
+                mod_roles = await self.bot.get_mod_roles(message.guild)
+                mod_mentions = " ".join(role.mention for role in mod_roles) if mod_roles else ""
+                
+                redirect_chain_status = self._get_redirect_chain_status(redirect_chain)
```

Ruff (0.8.2)

Local variable `redirect_chain_status` is assigned to but never used

Remove assignment to unused variable `redirect_chain_status`

(F841)